### ISSUE TITLE
License: MIT or BSD?

### ISSUE THREAD

**Issue by jhwohlgemuth** [spatialillusions/milsymbol#53]
Firstly, this is a neat project. Thank you for sharing :thumbsup:

On the topic of sharing, I have come across a minor issue regarding the license to be enforced on your library...

The [documentation and examples page](http://spatialillusions.com/milsymbol/index.html) says that the milsymbol.js library is BSD and [links to MIT license text](http://spatialillusions.com/milsymbol/license.txt).  The [README](https://github.com/spatialillusions/milsymbol/blob/master/README.md) in the GitHub repo says MIT and includes an [MIT license file](https://github.com/spatialillusions/milsymbol/blob/master/license.txt) (at least it is _titled as such_, I have not reviewed the license text as a lawyer would :smile:)

Some organizations are very particular about the wording in the license and might ignore this awesome library for no good reason.

Thanks again for sharing.

**Comment by spatialillusions** [comment]
Nice that you like my library. 

I can understand that there is some confusion, because the license used to be three clause BSD, but then I changed it to two clause MIT. I tried to update all information with this change, but unfortunately I missed some places. It is however the text in the license file that is the correct one and the one that should be followed. (And that is quite easy)

The three clause BSD and the two clause MIT is quite similar, but the three clause BSD says that you can't use the name of the library to promote your product, so you wouldn't be able to print a big banner saying "Our product uses correct military symbology, because we use the awesome library milsymbol." but this limitation has now been removed and you are free to print any banners you want.

To explain the license in very simple language it is do what you want in both open source and proprietary software, but keep my copyright and give credit somewhere in an about box or something. 

I have now seached all files on GitHub and updated them from BSD to MIT, so now it should be consistent. (The online documentation isn't updated yet, but will be so in the near future.)

If you have some lawyers that says that it needs to be under another license for you to be able to use it, please let me know and I will dual license it for you, the main idea is that it should be very easy to integrate milsymbol.

**Comment by spatialillusions** [comment]
After this commit I think everything should be changed. https://github.com/spatialillusions/milsymbol/commit/2234765880f0ff9ff8d430d915e7ed3fbe9a0873

**Comment by spatialillusions** [comment]
https://tldrlegal.com/license/mit-license
And in plain English what you can and can't do under MIT license.
